saleor/graphql/product/resolvers.py

```diff
@@ -21,6 +21,13 @@ def resolve_attributes(info, qs=None, in_category=None, in_collection=None, **_k
     return qs.distinct()
 
 
+def resolve_category_by_slug(info, slug):
+    try:
+        return models.Category.objects.get(slug=slug)
```

Comment:
Use `first()`, way faster and one line of code

Reply:
Done